README.md

```diff
@@ -459,44 +482,83 @@ them to the `'authproc'` array in your `metadata/saml20-idp-hosted.php` file.
 
 Example (for `metadata/saml20-idp-hosted.php`):
 
-    use Sil\PhpEnv\Env;
-    use Sil\Psr3Adapters\Psr3SamlLogger;
+```php
+use Sil\PhpEnv\Env;
+use Sil\Psr3Adapters\Psr3SamlLogger;
 
+$metadata['idp.example.com'] = [
     // ...
-
     'authproc' => [
-        10 => [
+        30 => [
```

Comment:
Why the change to 30?

Reply:
It's the sequence number we typically use in production.